client/core/bond.go

```diff
@@ -807,12 +809,14 @@ func (c *Core) PostBond(form *PostBondForm) (*PostBondResult, error) {
 	c.connMtx.RUnlock()
 	if found {
 		acctExists = !dc.acct.isViewOnly()
-		if acctExists && dc.acct.locked() { // require authDEX first to reconcile any existing bond statuses
-			return nil, newError(acctKeyErr, "acct locked %s (login first)", form.Addr)
-		}
-		if form.MaintainTier != nil || form.MaxBondedAmt != nil {
-			return nil, fmt.Errorf("maintain tier and max bonded amount may only be set when registering " +
-				"(use UpdateBondOptions to change bond maintenance settings)")
+		if acctExists {
+			if dc.acct.locked() { // require authDEX first to reconcile any existing bond statuses
+				return nil, newError(acctKeyErr, "acct locked %s (login first)", form.Addr)
+			}
+			if form.MaintainTier != nil || form.MaxBondedAmt != nil {
+				return nil, fmt.Errorf("maintain tier and max bonded amount may only be set when registering " +
+					"(use UpdateBondOptions to change bond maintenance settings)")
```

Comment:
whoops, yeah, these checks must all be under `acctExists`.  Missed this in view-only: https://github.com/decred/dcrdex/commit/02374f771e944b2f89c7ff81348571cb23456876#diff-6327e534faf5bc61df70b157ad5474582e26a4fd99bceb53316d758e9a5fe9bdR807